src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -154,7 +154,7 @@ function QuickbooksAdvancedPage({policy}: WithPolicyProps) {
                                             interactive={false}
                                         />
                                         <MenuItemWithTopDescription
-                                            title="Croissant Co Money in Clearing" // TODO: set to the current selected value
+                                            title={collectionAccountID}
```

Comment:
Is `collectionAccountID` actually the name of the account?
Let me check on this...

Reply:
yep, `collectionAccountID` is just an ID. You have to iterate over the actual accounts to get the title of the account 😄 

<img width="400" alt="Screenshot 2024-04-24 at 1 23 40 PM" src="https://github.com/Expensify/App/assets/98560306/27b5942d-4198-4062-bc59-18bc0838154e">